homeassistant/components/tile/device_tracker.py

```diff
@@ -84,6 +84,16 @@ def location_accuracy(self):
 
         Value in meters.
         """
+        if self._tile["last_tile_state"].get("h_accuracy") and not self._tile[
```

Comment:
Could you extract the variables first before checking. This is pretty unreadable.